packages/react-api/src/hooks/dataFilterExtensionUtils.js

```diff
@@ -55,34 +52,14 @@ function getUpdateTriggers(filtersWithoutTimeType, timeFilter) {
   };
 }
 
-function getFilterValue(filtersWithoutTimeType, timeFilter) {
-  const result = Array(MAX_GPU_FILTERS).fill(0);
-
-  // We evaluate all filters except the time filter using _buildFeatureFilter function.
-  // For the time filter, we return the value of the feature and we will change the getFilterRange result
-  // every time this filter changes
-  return (feature) => {
-    result[0] = _buildFeatureFilter({ filters: filtersWithoutTimeType, type: 'number' })(
-      feature
-    );
-
-    if (timeFilter) {
-      const f = feature.properties || feature;
-      result[1] = f[timeFilter.column];
-    }
-    return result;
-  };
-}
-
 // The deck.gl DataFilterExtension accepts up to 4 values to filter.
 // We're going to use the first value for all filter except the time filter
 // that will be managed by the second value of the DataFilterExtension
 export function getDataFilterExtensionProps(filters = {}) {
-  const { filtersWithoutTimeType, timeFilter } = getFiltersByType(filters);
+  const [filtersWithoutTimeType, timeFilter] = getFiltersByType(filters);
```

Comment:
Remove getFilterValue break the builder compatibility and move this function to a hook makes it not possible to use it in builder

Reply:
Updated.